apps/web/src/pages/CommunityInterests/CommunityTalentPage/CommunityTalentTable.tsx

```diff
@@ -566,6 +566,7 @@ const CommunityTalentTable = ({ title }: CommunityTalentTableProps) => {
         enableColumnFilter: false,
       },
     ),
+    // Update in #13855
     columnHelper.accessor(
```

Comment:
Added to the issue itself https://github.com/GCTC-NTGC/gc-digital-talent/issues/13855

<img width="960" height="334" alt="image" src="https://github.com/user-attachments/assets/40c99ca5-0bb4-4abc-bdd2-69aeefaee002" />